code/modules/mob/say.dm

```diff
@@ -10,7 +10,10 @@
 /mob/verb/whisper(message as text)
 	set name = "Whisper"
 	set category = "IC"
-	return
+	if(say_disabled)	//This is here to try to identify lag problems
+		usr << "<span class='danger'>Speech is currently admin-disabled.</span>"
+		return
+	say(message) //only carbons actually whisper, everything else just talks

```

Comment:
So things that try to whisper but aren't carbon mobs talk instead of saying nothing
this is kind of a stopgap fix but whisper is butts and I didn't look too hard at what i'd need to move to make it work at /mob